functions/sendMail.js

```diff
@@ -0,0 +1,26 @@
+const nodemailer = require('nodemailer')
+const functions = require('firebase-functions')
+
+exports.sendMail = (email, subject = `Hello world ${Date.now()}`) => {
+  try {
+    const transporter = nodemailer.createTransport({
+      host: functions.config().email.smtp,
+      port: 465,
+      secure: true,
+      auth: {
+        user: functions.config().email.user,
```

Comment:
these seem to be user/pass parameters to connect to a smtp server,  which sound really odd to me. I think it would be much better to use a service like sendgrid, as discussed in #15

Reply:
Sendgrid is fine, but I'm not willing to pay for a email sending service until we have a big demand for that, SMTP should be just fine while we just sending confirmation emails